vmc/resource_vmc_cluster.go

```diff
@@ -256,8 +259,10 @@ func resourceClusterRead(d *schema.ResourceData, m interface{}) error {
 			cluster["cluster_name"] = *clusterConfig.ClusterName
 			cluster["cluster_state"] = *clusterConfig.ClusterState
 			cluster["host_instance_type"] = *clusterConfig.EsxHostInfo.InstanceType
-			cluster["mssql_licensing"] = *clusterConfig.MsftLicenseConfig.MssqlLicensing
-			cluster["windows_licensing"] = *clusterConfig.MsftLicenseConfig.WindowsLicensing
+			if clusterConfig.MsftLicenseConfig != nil {
```

Comment:
I suppose this check is not related to the fix

Reply:
That is correct. While testing this fix, I came across a nil value being returned on resource creation when msft licensing config is not provided in the resource creation.

Reply:
Done